src/main/java/com/fundation/webservice/model/KeyFrameOfVideo.java

```diff
@@ -18,17 +19,12 @@
  * @author Josue Rodriguez
  * @version 1.0
  */
-public class KeyFrameOfVideo {
-    CriteriaKeyFrameVideo criteria;
-
-    public KeyFrameOfVideo(CriteriaKeyFrameVideo criteria) {
-        this.criteria = criteria;
-    }
-
+public class KeyFrameOfVideo implements IConvert{
     /**
      * Converts method get the keyframe depending of the number of frame.
```

Comment:
Maybe, I think to convert the format of this comment in a simple comment started with two slashes "//"

Reply:
Only in this case, this comment is for the method. I guess this way of commenting is standard javadoc style.

Reply:
The other class, i.e. VideoResponse has reverted its comment blocks back to single //s.
VideoResponse class is now fixed.